schema/annotation-source.yaml

```diff
@@ -161,30 +151,51 @@ $defs:
       A Statement (aka ‘Assertion’) represents a claim of purported truth as made by a particular agent, 
       on a particular occasion.
     heritable_properties:
+      target_proposition:
+        oneOf:
+        - $ref: "#/$defs/Proposition"
+        - $ref_curie: gks.core:CURIE
+        description: The Proposition about which the Statement is made.
+      evidence_direction:
+         type: string
+         enum: [ "supports", "disputes", inconclusive" ] 
+         description: >-
+           A term indicating the net direction of all evidence assessed against the target proposition. 
       evidence_level:
         $ref_curie: gks.core:Coding
         description: >- 
-          A term indicating the overall strength of support for the Statement based on all evidence assessed.
+          A term indicating the overall strength of support for or against the Statement's proposition, 
+          as provided by all evidence evaluated.
       evidence_score:
         $ref: "#/$defs/DataItem"
         description: >-
-          A quantitative score reflecting the overall strength of support for the Statement based on all 
-          evidence assessed.
-      target_proposition:
-        oneOf:
-        - $ref: "#/$defs/Proposition"
-        - $ref_curie: gks.core:CURIE
-        description: The Proposition about which the Statement is made.
-      conclusion:
+          A quantitative score reflecting the overall strength of support for or against the Statement's 
+          proposition, as provided by all evidence evaluated.
```

Comment:
As for evidence_level, refined definition to fit updated model / align with VA IM.